### PR TITLE
SONAR-23209 Move webapp build outside of gradle

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: .cirrus/cirrus-promote-webapp-assets.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  STATUS='it-passed-pr'
+# CirrusCI build on master branch
+elif [[ "$CIRRUS_CI" == "true" ]] && [[ ("$GITHUB_BRANCH" == "master" || "$GITHUB_BRANCH" == "branch-"*) ]]; then
+  PRIVATE_TARGET_REPO='sonarsource-private-builds'
+  PUBLIC_TARGET_REPO='sonarsource-public-builds'
+  STATUS='it-passed'
+fi
+
+if [ -n "${STATUS:-}" ]; then
+  echo "Promoting build sonarqube-webapp-assets#$BUILD_NUMBER"
```
Comment (david-cho-lerat-sonarsource) [inline .cirrus/cirrus-promote-webapp-assets.sh:15]: NP maybe the string "sonarqube-webapp-assets" could be stored in a variable as it's repeated multiple times?

File: gradle.properties [hunk 2]
```diff
@@ -5,7 +5,7 @@ version=10.8
 # 30 months from the release date for LTA versions
 # No change required for patch versions
 versionEOL=2025-05-27
-pluginApiVersion=10.13.0.2551
+pluginApiVersion=10.11.0.2468
```
Comment (david-cho-lerat-sonarsource) [inline gradle.properties:8]: was this change intentional?

File: .cirrus/cirrus-promote-webapp-assets.sh (hunk 1 above)
Comment (gregaubert) [reply .cirrus/cirrus-promote-webapp-assets.sh:15]: can you move that comment to the other repo, this PR is just there for validation and won't be merged.
